pages/fingerprints.md

```diff
@@ -0,0 +1,14 @@
+# CCR's public key fingerprints
+
+These are CCR's ssh public key fingerprints for login nodes:
+
+- `SHA256:QWv17p5d8CnZS7GPummIuLJW7vblXCWvMHlK7jrKtVA` (RSA)
+- `SHA256:6PDsSxEFoBaiPvOWyX8N281imkGaQV81ypJXtSOVX/k` (ECDSA)
+- `SHA256:QWv17p5d8CnZS7GPummIuLJW7vblXCWvMHlK7jrKtVA` (Ed25519)
+
+To avoid having to manually verify ssh keys for every CCR server you login to,
+you can add our ssh certificate to your `~.ssh/known_hosts`:  
```

Comment:
I like the addition of this page. Instead of 'your' here I recommend specifying that it's the file on their computer so they don't confuse it with the known_hosts file in their CCR account

```suggestion
you can add our ssh certificate to the `~/.ssh/known_hosts` on your personal device:  
```

Reply:
Agreed.. added

Reply:
You're just missing the / still before .ssh

Reply:
all fixed!